### ISSUE TITLE
[FEATURE] Builder sections are not translatable

### ISSUE THREAD

**Issue by tryallthethings** [AmruthPillai/Reactive-Resume#1109]
**Is your feature request related to a problem? Please describe.**

While reworking the German translation, I noticed that the default sections in the builder are not translatable as they're hard-coded in https://github.com/AmruthPillai/Reactive-Resume/blob/a687062866aa6741729369843c17520e8d5af4fc/server/src/resume/data/defaultState.ts?plain=1#L40

Examples:
- Work Experience
- Education
- Awards
- Certifications
- Publications
- Skills
- Languages

Applies to section headers, add new "section header name" and the entries shown on the CV itself

**Describe the solution you'd like**

Make default sections translatable. 😄

**Comment by MaxJa4** [comment]
Would be great if these section could be translatable (both in the sections and in the pdf itself, since it's the sections' default name).
Thanks!

**Comment by AmruthPillai** [comment]
> Would be great if these section could be translatable (both in the sections and in the pdf itself, since it's the sections' default name).
> Thanks!

The reason they're not translatable is because the section names can be edited by the user themselves to allow for maximum flexibility.

**Comment by tryallthethings** [comment]
> The reason they're not translatable is because the section names can be edited by the user themselves to allow for maximum flexibility.

True, but for a better user experience it would be great if they were pre-translated already.

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
